desc/geometry/curve.py

```diff
@@ -765,8 +766,17 @@ def __init__(
             errorif(knots[-1] > 2 * np.pi, ValueError, "knots must lie in [0, 2pi]")
             knots = knots[:-1] if closed_flag else knots
 
+        if discontinuous_indices is None:
+            interval_indices = [[]]
+        else:
+            interval_indices = [
+                [discontinuous_indices[i - 1], discontinuous_indices[i]]
```

Comment:
Is a check to ensure that the discontinuous indices are monotonic necessary? what if I put in [0,40, 10, 100, 50]? 

I'd think that would be good to add to avoid some weird curves being made that double back on themselves